packs/data/bestiary-effects.db/effect-guardians-aegis.json

```diff
@@ -0,0 +1,104 @@
+{
+    "_id": "AL7E03DYahfDhbcR",
+    "img": "systems/pf2e/icons/spells/spell-immunity.webp",
+    "name": "Effect: Guardian's Aegis",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p>The creature gains a +1 status bonus to saves against magical effects. The bonus increases to +2 if the effect originated from a demon or other chaotic evil fiend.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 5
+        },
+        "rules": [
+            {
+                "hideIfDisabled": true,
+                "key": "FlatModifier",
+                "predicate": [
+                    {
+                        "or": [
+                            "magical",
+                            "arcane",
+                            "divine",
+                            "primal",
+                            "occult"
+                        ]
+                    },
+                    {
+                        "not": {
+                            "or": [
+                                "origin:trait:demon",
+                                {
+                                    "and": [
+                                        "origin:trait:fiend",
+                                        "origin:trait:chaotic",
+                                        "origin:trait:evil"
+                                    ]
+                                }
+                            ]
+                        }
+                    }
+                ],
+                "selector": "saving-throw",
+                "slug": "guardians-aegis-1",
```

Comment:
Condense this to one simpler RE then use AdjustModifier to modify the bonus. It will end up being smaller overall because the predicates will be much simpler.

Reply:
Thanks for the tip, didn't know that was possible. Please have a close look at the rule elements, neber used AdjustModifier before.